apollo/__main__.py

```diff
@@ -2,11 +2,13 @@
 import json
 import logging
 import sys
+import pip
 from time import time
 
 from igraph import Graph
 from modelforge.logs import setup_logging
 from sourced.ml import extractors
+from sourced.ml.__main__ import ArgumentDefaultsHelpFormatterNoNone
```

Comment:
We need to move `ArgumentDefaultsHelpFormatterNoNone` to `sourced/ml/cmd_entries/args.py` and put it into `sourced/ml/cmd_entries/__init__.py`